docker-compose.yaml

```diff
@@ -56,15 +56,15 @@ services:
       - redis_data:/data/redis
 
   frequency:
-    image: frequencychain/standalone-node:v1.13.0-rc3
+    image: dsnp/instant-seal-node-with-deployed-schemas:latest
     # We need to specify the platform because it's the only image
     # built by Frequency at the moment, and auto-pull won't work otherwise
     platform: linux/amd64
     # Uncomment SEALING_MODE and SEALING_INTERVAL if you want to use interval sealing.
     # Other options you may want to add depending on your test scenario.
     environment:
-      - SEALING_MODE=interval
-      - SEALING_INTERVAL=12
+      - SEALING_MODE=instant
+      # - SEALING_INTERVAL=12
```

Comment:
We should avoid using instant seal in our test setups. But we can set the `SEALING_INTERVAL` to a lower number for test speed.